server/chat/backend/agent/tools/cloud_exec_tool.py

```
@@ -565,24 +602,38 @@ def setup_gcp_environment_isolated(user_id: str, selected_project_id: str | None
-        # BUILD ISOLATED ENVIRONMENT - NO global os.environ modification!
         isolated_env = {
             "PATH": os.environ.get("PATH", ""),
-            "HOME": "/home/appuser",  # Use terminal pod's writable home directory
+            "HOME": "/home/appuser",
             "USER": os.environ.get("USER", ""),
             "GOOGLE_OAUTH_ACCESS_TOKEN": access_token,
             "CLOUDSDK_AUTH_ACCESS_TOKEN": access_token,
             "GOOGLE_CLOUD_PROJECT": project_id,
-            "CLOUDSDK_AUTH_IMPERSONATE_SERVICE_ACCOUNT": sa_email,
-            "CLOUDSDK_IMPERSONATE_SERVICE_ACCOUNT": sa_email,
             "CLOUDSDK_CONFIG": "/tmp/.gcloud",
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/.gcloud"

(S108)